src/titlepane/TitlePane.ts

```diff
@@ -73,6 +79,30 @@ export class TitlePaneBase<P extends TitlePaneProperties = TitlePaneProperties>
 		}
 	}
 
+	protected animate(effects: any[]) {
```

Comment:
Is this `protected` for the purposes of extensibility?
I have raised a PR to type the `effects` array in the `meta`, https://github.com/dojo/widget-core/pull/865/files#diff-2b4688de89332ae4dfc392a34512c92aR36

Reply:
Yup, since things like duration and easing are no longer easily changed with CSS variables